MONGO.md

```diff
@@ -0,0 +1 @@
+mongodb+srv://mainUser:EJx3JFf4zh22F4kE@cluster0.qvt0x.mongodb.net/Fitigai?retryWrites=true&w=majority
```

Comment:
This is very insecure to push on git history, cause you basically expose your credential for your database publicly. (Especially that this repo is public cause we're working on an open-source code)